unpackaged/config/trial/flexipages/NPSP_Affiliation_Record_Page.flexipage

```diff
@@ -1,58 +1,39 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <FlexiPage xmlns="http://soap.sforce.com/2006/04/metadata">
     <flexiPageRegions>
-        <itemInstances>
```

Comment:
Should these files have been reverted back to  `.flexipage-meta.xml` filetypes? That's what I see in the git from before we pulled down the trial org metadata.

Reply:
Nevermind, I see that the API Version Update from 48 -> 53 is also included in the revert here.

Reply:
Yeah, it's confusing because the file that's on version 53 is now in config/trial_tso, this is just a copy of what was in the old config/trial folder, but it shows up as an update.